src/utils/chain/types.ts

```
@@ -164,7 +164,7 @@ export abstract class AbstractAsset implements IAsset {
     }
 
     toString(): string {
-        return `${this.printValue()} ${this.token.getSymbol()}`;
+        return `${this.printValue()} ${this.token?.getSymbol()}`;
```

token may be undefined? then maybe should return an error because it does not make sense to print the string without asset symbol.